server/base/src/main/java/org/apache/accumulo/server/util/FileUtil.java

```diff
@@ -163,15 +163,15 @@ private static Path createTmpDir(AccumuloConfiguration acuConf, VolumeManager fs
         try {
           if (reader != null)
             reader.close();
-        } catch (IOException e) {
+        } catch (Exception e) {
```

Comment:
Should these just catch RuntimeException, then? Or does this reader still throw checked exceptions, too?